extensions/amp-analytics/analytics-vars.md

```diff
@@ -90,7 +90,7 @@ Example value: `The New York Times - Breaking News, World News...`
 
 ### availableScreenHeight
 
-Provides the screen height in pixels available for the page rendering.
+Provides the screen height in pixels available for the page rendering. Note that this can be slightly more or less than the actual viewport height due to various browser quirks.
```

Comment:
If availableScreenWidth can have the same quirks, I'd advise writing about that in its section as well (for consistency).